app/builders/application_form_builder.rb

```diff
@@ -0,0 +1,78 @@
+class ApplicationFormBuilder < ActionView::Helpers::FormBuilder
+  include ActionView::Helpers::TagHelper
+
+  STYLES = {
+    default: "bg-transparent border border-2 border-white rounded-md text-white
+    placeholder-white/50 focus:bg-white focus:text-black focus:placeholder-gray
+    focus:border-white focus:ring-white transition-all",
+    toggle_field: "relative w-11 h-6 bg-gray-6 rounded-full peer dark:bg-gray-6
+    peer-checked:after:translate-x-full rtl:peer-checked:after:-translate-x-full
+    peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px]
+    after:start-[2px] after:bg-white after:rounded-full after:h-5 after:w-5
+    after:transition-all peer-checked:bg-red cursor-pointer",
+    error_message: "text-red-500 text-sm mt-1",
+    label: "font-bold text-base"
+  }.freeze
+
+  ERROR_STYLES = {
+    default: "ring-1 ring-red-500 !focus:ring-red-500"
+  }.freeze
+
+  [
+    :text_field,
+    :check_box,
+    :date_field,
+    :email_field,
+    :number_field,
+    :search_field,
+    :password_field,
+    :text_area
+  ].each do |method_name|
+    define_method(method_name) do |attribute, options = {}|
+      options = add_base_styles(options, method_name, attribute)
+      super(attribute, options)
+    end
+  end
+
+  def label(attribute, content_or_options = nil, options = {}, &block)
+    options = add_base_styles(options, __method__, attribute)
+    super
+  end
+
+  def select(attribute, choices = nil, options = {}, html_options = {})
+    options = add_base_styles(options, __method__, attribute)
+    super
+  end
+
+  def error_message(attribute, options = {})
```

Comment:
I suggest adding some basic styling so that other devs can use this helper

Reply:
I added basis styles 
![image](https://github.com/user-attachments/assets/3afef354-11a5-4d87-a9da-5ec668ddca01)